src/app/pokedex/pokedex-tile.css

```diff
@@ -0,0 +1,46 @@
+:root {
```

Comment:
could have a better file structure, dedicated css folder

Reply:
I'm uncertain how I'll organize things long term, but I'm open to the idea. For now, I'm thinking of having all components and CSS related to a concept (e.g. pokedex view) in a shared subdirectory. 

The most consistent best practice I could find was whatever made sense for the project